contracts/pcv/compound/CompoundPCVRouter.sol

```diff
@@ -65,9 +65,9 @@ contract CompoundPCVRouter is CoreRef {
     )
         external
         hasAnyOfThreeRoles(
-            TribeRoles.GOVERNOR,
-            TribeRoles.PCV_CONTROLLER,
-            TribeRoles.PCV_GUARD
+            VoltRoles.GOVERNOR,
+            VoltRoles.PCV_CONTROLLER,
+            VoltRoles.PCV_GUARD
         )
     {
         require(daiPSM.tin() == 0, "CompoundPCVRouter: maker fee not 0");
```

Comment:
lock after this line manually, unlock manually after `sellGem` so deposit succeeds